projects/observability/src/shared/components/cartesian/d3/scale/band/cartesian-band-scale.ts

```diff
@@ -40,4 +40,8 @@ export class CartesianBandScale<TData> extends CartesianScale<TData, string> {
   protected getEmptyScale(): ScaleBand<string> {
     return scaleBand().paddingInner(0.2).paddingOuter(0.1).align(1);
   }
+
+  public invert(x: number): number {
```

Comment:
this may need some changes.

Reply:
but not sure. Will check

Reply:
Band scale would return a string right? `CartesianBandScale<TData> extends CartesianScale<TData, string>`

so the domain is TData and range is a string. That means, the parameter for invert would be 'string', and return type would be TData.